lib/node_modules/@stdlib/stats/base/dnanmean/src/addon.c

```diff
@@ -0,0 +1,43 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include <node_api.h>
+#include "stdlib/napi/argv.h"
+#include "stdlib/napi/argv_int64.h"
+#include "stdlib/napi/argv_strided_float64array.h"
+#include "stdlib/napi/create_double.h"
+#include "stdlib/napi/export.h"
+#include "stdlib/stats/base/dnanmean.h"
```

Comment:
```suggestion
#include "stdlib/stats/base/dnanmean.h"
#include "stdlib/napi/argv.h"
#include "stdlib/napi/argv_int64.h"
#include "stdlib/napi/argv_strided_float64array.h"
#include "stdlib/napi/create_double.h"
#include "stdlib/napi/export.h"
#include <node_api.h>
```

The first include should always be the package's header file. Then stdlib deps. Then third party deps. Then built-in headers (e.g., `stdint.h`, etc).